bokehjs/src/lib/models/annotations/color_bar.ts

```diff
@@ -473,11 +474,15 @@ export class ColorBarView extends AnnotationView {
         start: 0,
         end: scale_length,
       }),
+        scan_result: this.model.color_mapper.scan_result,
     }
 
     switch (this.model.color_mapper.type) {
       case "LinearColorMapper": return new LinearScale(ranges)
       case "LogColorMapper":    return new LogScale(ranges)
+      case "EqHistColorMapper": {
+        return new LerpScale(ranges)
+      }
```

Comment:
`scan_result` is not a range, so it shouldn't go in an object called "ranges" Since it is specific to the linear interp scale, I would just pass it as an additional parameter after ranges.

Reply:
Also would prefer identical formatting to the two lines above